### PR TITLE
Issue #11028: Set the Secure attribute when SameSite=None

### LINKED ISSUE #11028: Ensure that if SameSite=None Secure is set automatically
If `<httpSession cookieSameSite="None"/>` the Secure attribute should be added automatically with out requiring `<httpSession cookieSecure="true"/>` to be set in the metaType.

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #11028

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#run-libby-bot

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_sameSite/server.xml [hunk 1]
```diff
@@ -22,17 +22,17 @@
...
+    <applicationManager autoExpand="true"/>
+
+    <jspEngine keepGenerated="true"/>
+
+    <logging
         traceSpecification="*=info=enabled:com.ibm.ws.session.*=all:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:HttpTransport=all:HTTPChannel=all:TCPChannel=all:GenericBNF=all:com.ibm.ws.jsp*=all:com.ibm.ws.session.*=all:com.ibm.ws.runtime.update.internal*=all:ChannelFramework=all"
-    	maxFileSize="20"
-    	maxFiles="10"
-    	traceFormat="BASIC"/>
+        maxFileSize="100"
```
Comment (mrsaldana) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_sameSite/server.xml]: do you foresee tracing files being an issue for this?

**Comment by pnicolucci** [comment]
@zhang-ca  that is correct if they want to use SameSite=None the Secure attribute will be enforced meaning a secure connection must be used.

**Comment by pnicolucci** [comment]
@mrsaldana  I plan to cut back the trace in another work item, currently I updated to 100MB size so we don't end up with 4-5 traces files that  that are 20mb each makes debugging a bit more difficult.

**Pull request merged**